README.md

```diff
@@ -293,6 +295,17 @@ With the newly added `fp8dqrow` scheme, the inference latency is **76.70 seconds
 |      5B      |   True    |   False    |     False      |    fp8dqrow    |         10.282 |             22.292 | 76.777  |
 |      5B      |   True    |    True    |     False      |    fp8dqrow    |         11.391 |               23.4 | 76.705  |
 
+**A100**
+
+|  model_type  |  compile  |  fuse_qkv  |  quantize_vae  |  quantization  |   model_memory |   inference_memory |   time  |
+|:------------:|:---------:|:----------:|:--------------:|:--------------:|:--------------:|:------------------:|:-------:|
+|      5B      |   False   |   False    |     False      |    fp6_e3m2    |          7.798 |             21.028 | 287.842 |
+|      5B      |   False   |    True    |     False      |    fp6_e3m2    |           8.63 |             23.243 | 285.294 |
+|      5B      |   False   |   False    |     False      |    fp5_e2m2    |          6.619 |              21.02 | 305.401 |
+|      5B      |   False   |    True    |     False      |    fp5_e2m2    |          7.312 |             23.237 | 304.725 |
+|      5B      |   False   |   False    |     False      |    fp4_e2m1    |          5.423 |             21.012 | 282.835 |
+|      5B      |   False   |    True    |     False      |    fp4_e2m1    |          5.978 |             23.228 | 280.262 |
+
 </details>
```

Comment:
Let's also include any interesting observations (like low latency and memory and good quality) if any.